qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -344,7 +344,7 @@
         </module>
         <module name="LineLength">
             <property name="max" value="80"/>
-            <property name="ignorePattern" value="^import .*$"/>
+            <property name="ignorePattern" value="^import .*$|^\s+\*.*https?:\/\/.+"/>
```

Comment:
@krzyk What about the case where HTML markup is used? Example:

```java
/**
 * @see <a href="http://google.com">http://google.com</a>
 */
```

The current regex in git will support that.